src/util.js

```diff
@@ -94,11 +93,11 @@ export function validateEntityType(model) {
 	}
 }
 
-export function truncateTables(Bookshelf, tables) {
-	return Promise.each(
-		tables,
-		(table) => Bookshelf.knex.raw(`TRUNCATE ${table} CASCADE`)
-	);
+export async function truncateTables(Bookshelf, tables) {
+	for (const table of tables) {
+		// eslint-disable-next-line no-await-in-loop
+		await Bookshelf.knex.raw(`TRUNCATE ${table} CASCADE`);
+	}
```

Comment:
In this case, we do need to do things sequentially - since some tables have foreign keys to others, the tables which reference need to be truncated before we can delete the tables that are referenced. For example, `editor` before `editor_type`. So I think this is a location where we should make an exception to the await-in-loop rule.

I've recently been looking at how we can make all the test data setup a bit better. Using https://www.npmjs.com/package/knex-db-manager might mean we can get rid of this truncate method and also preload data in a more consistent way.

Reply:
Thanks, that's what I was missing :)
So this seems like the right way to do it (with our current setup at least).